### ISSUE TITLE
Integrate new GP2X Wiz port

### ISSUE THREAD

**Issue by GoogleCodeExporter** [birdal46/grafx2#294]
```
Sourcecode here :
http://www.gp32spain.com/foros/downloads.php?do=file&id=2488
```

Original issue reported on code.google.com by `pulkoma...@gmail.com` on 23 Jan 2010 at 2:59

**Comment by GoogleCodeExporter** [comment]
```
I'm waiting on an update of the port to a more recent version (after the english
translation at least) so I can merge it efficiently.
```

Original comment by `pulkoma...@gmail.com` on 27 Jan 2010 at 12:40
- Added labels: **Milestone-NotPlanned**, **OpSys-Wiz**, **Type-Enhancement**
- Removed labels: **Milestone-**, **OpSys-**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I made an account and downloaded the archive.
Unfortunately they didn't keep the version.c, so it was trial and error to find 
the
revision they based on. It was r464.
A diff shows very little difference:
[ ] In realpath: #define PATH_MAX 32768
[ ] Resolution 320x240 for default resolution
[ ] On exit, it "returns to" the gp2x menu:
atexit(exitfunc);
(...)
static void exitfunc()
{
#ifdef __GP2X__
  chdir("/usr/gp2x");
  execl("/usr/gp2x/gp2xmenu", "/usr/gp2x/gp2xmenu", NULL);
#endif
}

```

Original comment by `yrizoud` on 22 Mar 2010 at 1:59

**Comment by GoogleCodeExporter** [comment]
```
ok... the first 2 are already done on gp2x, the third one is interesting and 
useful
(the menu is exited when launching an app so it lets more ram free for the app).

I'll integrate this later.
More interesting would be the makefile; but I think they used a code::blocks 
project
instead...
```

Original comment by `pulkoma...@gmail.com` on 22 Mar 2010 at 3:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `pulkoma...@gmail.com` on 9 Aug 2010 at 9:47
- Added labels: **Com**

**Comment by GoogleCodeExporter** [comment]
Original comment by `pulkoma...@gmail.com` on 9 Aug 2010 at 9:47
- Added labels: **Component-BuildSys**
- Removed labels: **Com**

**Comment by GoogleCodeExporter** [comment]
```
http://dl.openhandhelds.org/cgi-bin/caanoo.cgi?0,1,1,0,113,513

Port to the Caanoo. Should be merged too. It is based on version 2.2 so should 
be less difficult to merge back than the Wiz one.
```

Original comment by `pulkoma...@gmail.com` on 22 Sep 2010 at 11:27

**Comment by GoogleCodeExporter** [comment]
```
I checked the differences (compared with the gp2x package), it's mostly a 
matter of different joystick buttons and some different #defines for detection, 
ie no longer _gp2x_.
I need the sdk to understand better the #defines. It's not the "GPH SDK" v10.2 
or v10.5, which should only recognize up to the Wiz (?) hopefully it will be 
this one:
http://dl.openhandhelds.org/cgi-bin/caanoo.cgi?0,0,0,0,17,477

```

Original comment by `yrizoud` on 23 Sep 2010 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
Seems this is working ok now.
```

Original comment by `pulkoma...@gmail.com` on 15 Dec 2010 at 4:23
- Changed state: **Fixed**
